modules/administration-guide/partials/proc_sharing-che-permissions.adoc

```diff
@@ -3,13 +3,20 @@
 [id="sharing-{prod-id-short}-permissions_{context}"]
 = Sharing {prod-short} permissions
 
-A user with *setPermissions* privileges can share a workspace and grant `read`, `use`, `run`, `configure`, or `setPermissions` privileges for other users.
+If you have the *setPermissions* user privileges, you can share a workspace with other users and grant the users any of the following workspace permissions:
```

Comment:
@themr0c The procedure has no mention of any of the workspace permissions. Are all of them granted automatically as part of sharing a workspace? If so, then we need to mention this fact.